projects/cryptography/symenc_fuzz.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python3
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import sys
+import base64
+import atheris
+with atheris.instrument_imports():
+    from cryptography.fernet import Fernet
+    from cryptography.hazmat.primitives import hashes
+    from cryptography.hazmat.primitives.kdf.pbkdf2 import PBKDF2HMAC
+
+def TestInput(input_bytes):
+   fdp = atheris.FuzzedDataProvider(input_bytes)
+
+   key = Fernet(Fernet.generate_key())
+   token = key.encrypt(fdp.ConsumeBytes(32))
+   text = key.decrypt(token)
+
+   password = fdp.ConsumeBytes(8)
+   salt = fdp.ConsumeBytes(16)
+   kdf = PBKDF2HMAC(
+      algorithm=hashes.SHA256(),
+      length=32,
+      salt=salt,
+      iterations=390000,
```

Comment:
Using a high iteration will make this slow, and doesn't improve the coverage here. At a minimum I'd set `iterations=1`.